helm/csi-powermax/templates/node.yaml

```diff
@@ -177,6 +177,20 @@ spec:
               value: "{{ .Values.node.topologyControl.enabled }}"
             {{- end }}
             {{- end }}
+            - name: X_CSI_VSPHERE_ENABLED
+              value: "{{ .Values.vSphere.enabled }}"
+            {{- if eq .Values.vSphere.enabled true }}
+            - name: X_CSI_VSPHERE_PORTGROUP
+              value: {{ required "Must provide portgroup for vsphere" .Values.vSphere.fcPortGroup }}
```

Comment:
Same comment. update the required message

Reply:
updated message.